statsmodels/genmod/generalized_linear_model.py

```diff
@@ -281,7 +320,8 @@ def loglike(self, params, scale=None):
         expval = self.family.link.inverse(lin_pred)
         if scale is None:
             scale = self.estimate_scale(expval)
-        return self.family.loglike(expval, self.endog, scale)
+        return self.family.loglike(expval, self.endog, self.freq_weights,
+                                   scale)
```

Comment:
This just seemed more logical order to me. Happy to change it otherwise.

> On Mar 8, 2016, at 6:33 PM, Josef Perktold <notifications@github.com> wrote:
> 
> In statsmodels/genmod/generalized_linear_model.py:
> 
> > @@ -281,7 +320,8 @@ def loglike(self, params, scale=None):
> >          expval = self.family.link.inverse(lin_pred)
> >          if scale is None:
> >              scale = self.estimate_scale(expval)
> > -        return self.family.loglike(expval, self.endog, scale)
> > +        return self.family.loglike(expval, self.endog, self.freq_weights,
> > +                                   scale)
> adding freq_weights before scale is not backwards compatible.
> does it matter?
> 
> —
> Reply to this email directly or view it on GitHub.
>